docs/ruleset_functions/get_zones_hvac_system_humid_p.md

```diff
@@ -0,0 +1,41 @@
+## get_zones_hvac_system_humid_p
+
+**Description:** Get the list of P_RMR zones and corresponding HVAC system IDs where humidification is applicable.
+
```

Comment:
I'm assuming this description is "Get the list of P_RMR zones and corresponding HVAC system IDs where humidification has been modeled"? not sure what we mean by "humidification is applicable"

Reply:
Text has been updated. Good point.